components/Settings/FavoritesTab.js

```diff
@@ -1,14 +1,14 @@
 import ClubTableRow from '../ClubTableRow'
 
 export default (props) => {
-  const { updateFavorites, favorites, clubs } = props
+  const { keyword, updateFavorites, favorites, clubs } = props
 
   const findClub = clubs ? code => clubs.find(club => club.code === code) : () => {}
 
   if (!favorites || !favorites.length) {
     return (
       <p className="has-text-light-grey" style={{ paddingTop: 200, textAlign: 'center' }}>
-        No bookmarks yet! Browse clubs <a href="/">here.</a>
+        No {keyword}s yet! Browse clubs <a href="/">here.</a>
```

Comment:
Can you give an example of a keyword? a bit confused of the use here / how we phrase it

Reply:
Ah keyword is either `bookmark` or `subscription`. I reused the FavoritesTab for both, since in this case both the bookmark and subscribe tab are functionally equivalent.